src/base/object_representation_sr.c

```diff
@@ -1894,13 +1964,20 @@ or_install_btids_class (OR_CLASSREP * rep, BTID * id, DB_SEQ * constraint_seq, i
       er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_OUT_OF_VIRTUAL_MEMORY, 1, sizeof (OR_ATTRIBUTE *) * att_cnt);
       return;
     }
+#ifdef SERVER_MODE
+  index->att_cnt = att_cnt;
```

Comment:
Is it fixing a missing part?

Reply:
It is for getting size of the structures when they are freed.